debugtools/DDR_VM/src/com/ibm/j9ddr/CompatibilityConstants29.dat

```diff
@@ -59,7 +59,7 @@ J9FieldFlags.J9FieldFlagIsNullRestricted = 0
 J9JavaAccessFlags.J9AccClassIsUnmodifiable = 0
 J9JavaAccessFlags.J9AccRecord = 0
 J9JavaAccessFlags.J9AccSealed = 0
-J9JavaAccessFlags.J9AccValueType = 0
+J9JavaAccessFlags.J9AccClassHasIdentity = 0
```

Comment:
`J9AccClassHasIdentity` should appear before `J9AccClassIsUnmodifiable` to retain order.